Automattic/wp-calypso
# NUX: integrate preview response from verticals endpoint

## ⚠️ Requires `D23034-code`

## Changes proposed in this Pull Request

Taking the preview content data kindly provided by the Verticals API and plugging it into the site preview component.

<img width="994" alt="screen shot 2019-01-16 at 3 02 06 pm" src="https://user-images.githubusercontent.com/6458278/51225711-e5f45400-199f-11e9-81c9-3fa1cf12b904.png">

- [ ] Answer: Is there a better place to store the preview content other than in the state?
   * Store in `localStorage/sessionStorage`, and/or make an API request if we have a vertical cached?
   * We delete the signup data anyway when we dispatch `SIGNUP_COMPLETE_RESET`.
- [x] _Will there be site style options?_ Coming as we transition over to Twenty Nineteen

## Dependencies

We need to publish `D23034-code` before we can merge this PR.

## Testing instructions

1. ⚠️ Requires `D23034-code`
2. Fire up this branch and head to http://calypso.localhost:3000/start/onboarding-dev/site-type
3. Test various vertical results at http://calypso.localhost:3000/start/onboarding-dev/site-topic-with-preview
4. Enter site information, and make sure the values are updated in the preview code

<img width="984" alt="screen shot 2019-01-17 at 11 59 43 am" src="https://user-images.githubusercontent.com/6458278/51288253-6f169400-1a4f-11e9-9dcc-c7c0aeca5f94.png">
